### PR TITLE
Create new QueueItem for proper search.

### PR DESCRIPTION
**Pull request by p0psicles**
ProperSearches are currently not running threaded, making it block other operations.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

#### TODO:
- [x] Make sure manage - searches, shows poper search is running
- [x] Vueify server status, show proper search is running
- [x] Fix subtitle search status in manage - searches

@medariox the proper search was running directly in the scheduler. Which isn't treaded. This makes it block other requests like the apiv2. Moving it into a QueueItem fixes that.

### CODE REVIEW

**Comment by XxUnkn0wnxX** [comment]
proper search kept looping for 8+ hours saw in the logs for all shows same EP, same provider like 6+ times.
had to restart medusa to just stop it from searching forever... something keeps restarting it.

**Comment by p0psicles** [comment]
Tnx

File: medusa/server/web/home/handler.py [hunk 1]
```diff
@@ -579,22 +579,29 @@ def getPushbulletDevices(api=None):
...
-                root_dir[subject] = helpers.get_disk_space_usage(subject)
+        [Converted to VueRouter]
+        """
+        return PageTemplate(rh=self, filename='index.mako').render()
 
-        t = PageTemplate(rh=self, filename='status.mako')
-        return t.render(title='Status', header='Status',
-                        tvdirFree=tv_dir_free, rootDir=root_dir,
-                        controller='home', action='status')
+        # tv_dir_free = helpers.get_disk_space_usage(app.TV_DOWNLOAD_DIR)
```
Comment (p0psicles) [inline medusa/server/web/home/handler.py:589]: Reminder

File: setup.py [hunk 2]
```diff
@@ -3,13 +3,74 @@
...
+import subprocess
 import sys
 
 from setuptools import setup
+from setuptools.command.install import install
 from setuptools.command.test import test as TestCommand
 
 here = os.path.abspath(os.path.dirname(__file__))
 
+# These requirements probably won't be needed
```
Comment (p0psicles) [inline setup.py:15]: Remove all this. Play around in new pr

File: tests/conftest.py [hunk 3]
```diff
@@ -43,6 +43,13 @@ def pytest_collection_modifyitems(config, items):
                 items.remove(item)
 
 
+def pytest_configure(config):
```
Comment (p0psicles) [inline tests/conftest.py:46]: Remove this

**Comment by p0psicles** [comment]
@medariox I'll clean this up a little. Merge in develop. And then where good

**Pull request merged**